### ISSUE TITLE
Error during installation

### ISSUE THREAD

**Issue by SampicBE** [mfrachet/rn-placeholder#87]
Hello,

When I add your package to my application, I can not install it because an error occurs during the installation.

Your postinstall obviously blocks my installation, it goes without saying that I have yarn on my post :)

Can you do the right thing please?

Thank you

```bash
$ yarn --cwd ./example
'yarn' n'est pas reconnu en tant que commande interne
ou externe, un programme ex�cutable ou un fichier de commandes.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by mfrachet** [comment]
Thanks for the report and sorry about that 😓 

This should be fixed on `v3.0.0-alpha.2` in this pr https://github.com/mfrachet/rn-placeholder/pull/88.

It has just been published. Let me know if you're still in trouble 😊

**Comment by SampicBE** [comment]
Everything is good, thank you for the responsiveness !
